scripts/resetAll.sh

```diff
@@ -47,6 +47,12 @@ kubectl apply -f https://github.com/knative/operator/releases/download/knative-v
 kubectl wait --for=condition=ready pod -l app.kubernetes.io/name=knative-operator
 
 CACERT=$dir/registry/share/out/ca.cert.pem
+
+
```

Comment:
What is this change for? Whatever it is for, it probably deserves to be a separate PR because I think it's surely quite unrelated to log tags.

Reply:
this does not install knative because it tries to use the certificate and fails. By adding this line we could make sure that it will install knative even if it fails to use the certificate